OAuthSwift/OAuthSwiftClient.swift

```diff
@@ -10,15 +10,30 @@ import Foundation
 
 var OAuthSwiftDataEncoding: NSStringEncoding = NSUTF8StringEncoding
 
+public protocol OAuthSwiftRequestHandle {
+    var requestConfig: OAuthSwiftHTTPRequestConfig {get}
+    func cancel()
+}
+
 public class OAuthSwiftClient: NSObject {
 
-    private(set) public var credential: OAuthSwiftCredential
+    public var credential: OAuthSwiftCredential
```

Comment:
why set public credential? 
(all field are not serialized, this could add a bug if someone set the credential, the protocol version will be 1)